### PR TITLE
AFC-1710 Added activity_field to Dev-12-branch

### PR DESCRIPTION
**Pull request by fredrikarvas-vertel**

### CODE REVIEW

File: activity_field/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Activity Tab DEPRECATED',
```
Comment (arturchilangwa-vertel) [inline activity_field/__manifest__.py:23]: Why does it the name mention `depricated` but the module is still installable?

**Review by arturchilangwa-vertel — CHANGES_REQUESTED** [review]

File: activity_field/__manifest__.py (hunk 1 above)
Comment (fredrikarvas-vertel) [reply activity_field/__manifest__.py:23]: It means it should not be installed to CRM at this point.

**Comment by arturchilangwa-vertel** [comment]
@fredrikarvas-vertel 
Im getting this error when I try to install the module. 
Unfortunately, the error does not refer to any obviuos modules installed in `/usr/share/odoo*`
```
2021-01-25 13:34:43,645 3501 ERROR crm4 odoo.modules.registry: Failed to load registry 
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/odoo/modules/registry.py", line 86, in new
    odoo.modules.load_modules(registry._db, force_demo, status, update_module)
  File "/usr/lib/python3/dist-packages/odoo/modules/loading.py", line 421, in load_modules
    loaded_modules, update_module, models_to_check)
  File "/usr/lib/python3/dist-packages/odoo/modules/loading.py", line 313, in load_marked_modules
    perform_checks=perform_checks, models_to_check=models_to_check
  File "/usr/lib/python3/dist-packages/odoo/modules/loading.py", line 194, in load_module_graph
    registry.setup_models(cr)
  File "/usr/lib/python3/dist-packages/odoo/modules/registry.py", line 275, in setup_models
    model._setup_fields()
  File "/usr/lib/python3/dist-packages/odoo/models.py", line 2656, in _setup_fields
    field.setup_full(self)
  File "/usr/lib/python3/dist-packages/odoo/fields.py", line 520, in setup_full
    self._setup_related_full(model)
  File "/usr/lib/python3/dist-packages/odoo/fields.py", line 563, in _setup_related_full
    field = target._fields[name]
KeyError: 'office'
2021-01-25 13:34:43,645 3501 ERROR crm4 odoo.http: Exception during JSON request handling. 
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 656, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 314, in _handle_exception
    raise pycompat.reraise(type(exception), exception, sys.exc_info()[2])
  File "/usr/lib/python3/dist-packages/odoo/tools/pycompat.py", line 87, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 698, in dispatch
    result = self._call_function(**self.params)
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 346, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/usr/lib/python3/dist-packages/odoo/service/model.py", line 98, in wrapper
    return f(dbname, *args, **kwargs)
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 339, in checked_call
    result = self.endpoint(*a, **kw)
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 941, in __call__
    return self.method(*args, **kw)
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 519, in response_wrap
    response = f(*args, **kw)
  File "/usr/lib/python3/dist-packages/odoo/addons/web/controllers/main.py", line 966, in call_button
    action = self._call_kw(model, method, args, {})
  File "/usr/lib/python3/dist-packages/odoo/addons/web/controllers/main.py", line 954, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/usr/lib/python3/dist-packages/odoo/api.py", line 759, in call_kw
    return _call_kw_multi(method, model, args, kwargs)
  File "/usr/lib/python3/dist-packages/odoo/api.py", line 746, in _call_kw_multi
    result = method(recs, *args, **kwargs)
  File "<decorator-gen-61>", line 2, in button_immediate_install
  File "/usr/lib/python3/dist-packages/odoo/addons/base/models/ir_module.py", line 74, in check_and_log
    return method(self, *args, **kwargs)
  File "/usr/lib/python3/dist-packages/odoo/addons/base/models/ir_module.py", line 445, in button_immediate_install
    return self._button_immediate_function(type(self).button_install)
  File "/usr/lib/python3/dist-packages/odoo/addons/base/models/ir_module.py", line 561, in _button_immediate_function
    modules.registry.Registry.new(self._cr.dbname, update_module=True)
  File "/usr/lib/python3/dist-packages/odoo/modules/registry.py", line 86, in new
    odoo.modules.load_modules(registry._db, force_demo, status, update_module)
  File "/usr/lib/python3/dist-packages/odoo/modules/loading.py", line 421, in load_modules
    loaded_modules, update_module, models_to_check)
  File "/usr/lib/python3/dist-packages/odoo/modules/loading.py", line 313, in load_marked_modules
    perform_checks=perform_checks, models_to_check=models_to_check
  File "/usr/lib/python3/dist-packages/odoo/modules/loading.py", line 194, in load_module_graph
    registry.setup_models(cr)
  File "/usr/lib/python3/dist-packages/odoo/modules/registry.py", line 275, in setup_models
    model._setup_fields()
  File "/usr/lib/python3/dist-packages/odoo/models.py", line 2656, in _setup_fields
    field.setup_full(self)
  File "/usr/lib/python3/dist-packages/odoo/fields.py", line 520, in setup_full
    self._setup_related_full(model)
  File "/usr/lib/python3/dist-packages/odoo/fields.py", line 563, in _setup_related_full
    field = target._fields[name]
KeyError: 'office'
```

**Comment by fredrikarvas-vertel** [comment]
På min maskin får jag det här felet, men då är modulen: hr_office installerad.

Fel:
Odoo Server Error

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 656, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 314, in _handle_exception
    raise pycompat.reraise(type(exception), exception, sys.exc_info()[2])
  File "/usr/lib/python3/dist-packages/odoo/tools/pycompat.py", line 87, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 698, in dispatch
    result = self._call_function(**self.params)
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 346, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/usr/lib/python3/dist-packages/odoo/service/model.py", line 98, in wrapper
    return f(dbname, *args, **kwargs)
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 339, in checked_call
    result = self.endpoint(*a, **kw)
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 941, in __call__
    return self.method(*args, **kw)
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 519, in response_wrap
    response = f(*args, **kw)
  File "/usr/share/core-odoo/addons/web/controllers/main.py", line 967, in call_button
    action = self._call_kw(model, method, args, {})
  File "/usr/share/core-odoo/addons/web/controllers/main.py", line 955, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/usr/lib/python3/dist-packages/odoo/api.py", line 759, in call_kw
    return _call_kw_multi(method, model, args, kwargs)
  File "/usr/lib/python3/dist-packages/odoo/api.py", line 746, in _call_kw_multi
    result = method(recs, *args, **kwargs)
  File "<decorator-gen-61>", line 2, in button_immediate_install
  File "/usr/share/core-odoo/addons/base/models/ir_module.py", line 74, in check_and_log
    return method(self, *args, **kwargs)
  File "/usr/share/core-odoo/addons/base/models/ir_module.py", line 445, in button_immediate_install
    return self._button_immediate_function(type(self).button_install)
  File "/usr/share/core-odoo/addons/base/models/ir_module.py", line 561, in _button_immediate_function
    modules.registry.Registry.new(self._cr.dbname, update_module=True)
  File "/usr/lib/python3/dist-packages/odoo/modules/registry.py", line 86, in new
    odoo.modules.load_modules(registry._db, force_demo, status, update_module)
  File "/usr/lib/python3/dist-packages/odoo/modules/loading.py", line 421, in load_modules
    loaded_modules, update_module, models_to_check)
  File "/usr/lib/python3/dist-packages/odoo/modules/loading.py", line 313, in load_marked_modules
    perform_checks=perform_checks, models_to_check=models_to_check
  File "/usr/lib/python3/dist-packages/odoo/modules/loading.py", line 194, in load_module_graph
    registry.setup_models(cr)
  File "/usr/lib/python3/dist-packages/odoo/modules/registry.py", line 275, in setup_models
    model._setup_fields()
  File "/usr/lib/python3/dist-packages/odoo/models.py", line 2656, in _setup_fields
    field.setup_full(self)
  File "/usr/lib/python3/dist-packages/odoo/fields.py", line 520, in setup_full
    self._setup_related_full(model)
  File "/usr/lib/python3/dist-packages/odoo/fields.py", line 563, in _setup_related_full
    field = target._fields[name]
KeyError: 'user_id'

**Comment by fredrikarvasarbetsformedlingen** [comment]
This module is replaced with partner_activity

**Pull request closed without merging**